docs/UserGuide.md

```diff
@@ -123,7 +123,7 @@ Examples:
 
 Edits an existing customer in ReadyBakey's contact list.
 
-Format: `edito INDEX [d/DETAILS] [c/DELIVERYDATETIME] [m/COLLECTION_TYPE] [r/REMARKS]`
+Format: `edito INDEX [d/DETAILS]… [c/DELIVERYDATETIME] [m/COLLECTION_TYPE] [r/REMARKS]`
```

Comment:
We can put the details at the back to make the ... more prominent?

Reply:
Got it, will move it